service/prometheus/prometheus.go

```diff
@@ -1,27 +1,15 @@
 package prometheus
 
 import (
-	"strings"
-
 	"k8s.io/client-go/pkg/api/v1"
 )
 
 const (
-	CertificateAnnotation = "giantswarm.io/prometheus-config-controller/certificate"
-	ClusterAnnotation     = "giantswarm.io/prometheus-config-controller/cluster"
+	ClusterAnnotation = "giantswarm.io/prometheus-config-controller/cluster"
 )
 
-// GetCertificateName returns the value of the certificate annotation, splitted by '/'.
-// Assumed that the service contains this annotation, see `FilterInvalidServices`.
-func GetCertificateName(service v1.Service) (string, string) {
```

Comment:
Instead of specifying a secret with the certificate annotation, we use the cluster id specified in the cluster annotation to match the cluster id label, which is much nicer and more aligned.